### PR TITLE
Redesign GUI to reduce clutter

### LINKED ISSUE #54: Implement UI element to display delivery information
The user should be able to view delivery information clearly.

### PR DESCRIPTION
**Pull request by Skeltons**
closes #54 
Might change color scheme later on
image

### CODE REVIEW

File: src/main/resources/view/TestWindow.fxml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                                    <padding>
+                                        <Insets top="5" right="10" bottom="5" left="10" />
+                                    </padding>
+                                </StackPane>
+
+                                <StackPane VBox.vgrow="NEVER" fx:id="commandBoxPlaceholder" styleClass="pane-with-border">
+                                    <padding>
+                                        <Insets top="5" right="10" bottom="5" left="10" />
+                                    </padding>
+                                </StackPane>
```
Comment (Mehvin) [inline src/main/resources/view/TestWindow.fxml:58]: I like the overall redesign of the GUI particularly how the command box is at the bottom now!

File: src/main/java/bookopedia/ui/PersonCard.java [hunk 2]
```diff
@@ -54,26 +51,8 @@ public PersonCard(Person person, int displayedIndex) {
...
-                    Label label = new Label(parcelIndex.incrementAndGet() + ". " + parcel.parcelName);
-                    if (parcel.isFragile() && parcel.isBulky()) {
-                        label.getStyleClass().add("isFragileAndBulky");
-                    } else if (parcel.isFragile()) {
-                        label.getStyleClass().add("isFragile");
-                    } else if (parcel.isBulky()) {
-                        label.getStyleClass().add("isBulky");
-                    }
-                    parcels.getChildren().add(label);
-                });
```
Comment (Mehvin) [inline src/main/java/bookopedia/ui/PersonCard.java:73]: Maybe as a possible alternative, we can leave the parcels in the person list card at the right side too?

**Review by Mehvin — APPROVED** [review]
Provided some suggestions, but other than that all LGTM!

File: src/main/java/bookopedia/ui/PersonCard.java (hunk 2 above)
Comment (Skeltons) [reply src/main/java/bookopedia/ui/PersonCard.java:73]: Ok sure

**Pull request merged**